lib/beaker/host/unix/exec.rb

```diff
@@ -101,6 +101,16 @@ def touch(file, abs=true)
     (abs ? '/bin/touch' : 'touch') + " #{file}"
   end
 
+  # Update ModifiedDate on a file
+  # @param [String] file Path to the file
+  # @param [String] timestamp Timestamp to set
+  def modified_at(file, timestamp = nil)
+    require 'date'
+    time = timestamp ? DateTime.parse("#{timestamp}") : DateTime.now
```

Comment:
Do we care if timestamp can't be parsed, or we let beaker explode?

Reply:
Explode sounds so drastic, it's true, it will raise an exception, but that would have happened even with an invalid cli command. 
That's why I used DateTime, it's more flexible when parsing, compared to Time.

Reply:
yes 😅